git_connector.py

```diff
@@ -440,7 +449,17 @@ def _git_pull(self, param):
     def _delete_clone(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
 
-        repo_dir = self.app_state_dir / self.repo_name
+        if not param.get("repo_url", None):
+            if not self.repo_name:
+                msg = "You must provide the clone URL for a previously cloned repo or have provided repo information in the app configuration."
+                return action_result.set_status(phantom.APP_ERROR, msg)
+            repo_dir = self.app_state_dir / self.repo_name
+        else:
+            self.modified_repo_uri = param.get("repo_url")
+            folder_name = urllib.parse.urlparse(self.modified_repo_uri).path.split(".")[0].replace("/", "_")
```

Comment:
We should also add branch parameter for deleting the repo, as cloning the repo appends the branch at the end of folder name. 

So during deleting repo with current implementation throes error stating "repo could not be found".